bugsnag-android-core/src/main/java/com/bugsnag/android/SessionTracker.java

```diff
@@ -36,6 +36,7 @@ class SessionTracker extends BaseObservable implements ForegroundDetector.OnActi
     private volatile Session currentSession = null;
     final BackgroundTaskService backgroundTaskService;
     final Logger logger;
+    private boolean shouldSuppressFirstAutoSession = true;
```

Comment:
Given that this *only* happens when the first session is manual, maybe this field should be named something more like: `firstAutoSessionSuppressed` and rather start as `false`?